ansible/playbooks/AdoptOpenJDK_AIX_Playbook/roles/rbac/tasks/main.yml

```diff
@@ -0,0 +1,34 @@
+#########################################################################
+# Playbook to create authorization to access realtime clock privelidge
+# The role is assigned to the jenkins user
```

Comment:
Given that this is quite unusual, can we add a link in the comments to the issue regarding this problem, so that people can understand why we added it.